Disable ming support.
ming is for generating Flash files. Flash was discontinued in 2020.
https://github.com/autotrace/autotrace/issues/137
--- configure.ac.orig	2024-01-29 15:37:09.000000000 -0600
+++ configure.ac	2024-05-20 12:12:43.000000000 -0500
@@ -158,10 +158,7 @@
 dnl
 
 HAVE_LIBSWF=no
-AC_CHECK_HEADER(ming.h,
-	        swf_header_found=yes,
-		swf_header_found=no)
-if test "${swf_header_found}" = yes ; then
+if false; then
 	PKG_CHECK_MODULES([LIBSWF],
 			  [libming],
 			  [
